src/main/scala/org/zalando/nakadi/client/actor/PartitionReceiverActor.scala

```diff
@@ -75,6 +77,7 @@ class PartitionReceiver (val endpoint: URI,
 
 
   // TODO check earlier ListenParameters
+  private
```

Comment:
I also thought about declaring these methods private but they cannot be accessed via the ActorRef anyway